### PR TITLE
Vero destination documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Vero destination documentation

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Vero destination docs](https://www.notion.so/rudderstacks/Vero-Integration-Documentation-69283c08d2e1441c91d5f49173587ae5)

### CODE REVIEW

**Review by yashasvibajpai — APPROVED** [review]

**Comment by yashasvibajpai** [comment]
@sauravlal-rudderstack Can you please check with the latest web device mode changes to be in parity with the doc here

**Review by sandeepdsvs — APPROVED** [review]

**Review by sauravlal-rudderstack — APPROVED** [review]

**Comment by sauravlal-rudderstack** [comment]
> @sauravlal-rudderstack Can you please check with the latest web device mode changes to be in parity with the doc here

Done

**Comment by ameypv-rudder** [comment]
@yashasvibajpai @sauravlal-rudderstack I think Saurav mentioned in the closed PR that the `tags` functionality is applicable for `alias` calls as well. Can you please confirm? Only this update is pending.

**Comment by sauravlal-rudderstack** [comment]
>

File: docs/destinations/streaming-destinations/vero/device-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+### Supported mappings
+
+The following table lists the mapping of the RudderStack traits to the corresponding Vero properties:
+
+| RudderStack trait | Vero property | Presence |
+| :-----| :------| :-------|
+| `userId`/`anonymousId` | `id` | Required, if `email` is not present. |
+| `email` | `email` | Required, if `userId`/`anonymousId` is not present. |
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/vero/device-mode.mdx:30]:
```suggestion
| `email` | `email` | Optional |
```

File: docs/destinations/streaming-destinations/vero/device-mode.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+| `context.traits`/`traits` | `traits` |  Optional |
+
+<div class="infoBlock">
+If the event includes <code class="inline-code">email</code> and no <code class="inline-code">userId</code>, Vero will automatically use <code class="inline-code">email</code> as the <code class="inline-code">id</code>.
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/vero/device-mode.mdx:34]: Same as cloud mode.

File: docs/destinations/streaming-destinations/vero/cloud-mode.mdx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+| `traits` | `data` |  Optional |
+| `context.os.name` | `channels.platform` | Optional |
+| `context.device.token` | `channels.address` | Optional |
+| `context.device.name` | `channels.device` | Optional |
+| `context.timezone` | `timezone` | Optional |
+| `context.locale`| `locale` | Optional |
+| `context.userAgent` | `userAgent` | Optional |
+
+<div class="infoBlock">
+If the event includes <code class="inline-code">email</code> and no <code class="inline-code">userId</code>, Vero will automatically use <code class="inline-code">email</code> as the <code class="inline-code">id</code>.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/vero/cloud-mode.mdx]: @ItsSudip this is incorporated. Please have a look once and let me know if we're good.

File: docs/destinations/streaming-destinations/vero/cloud-mode.mdx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+```javascript
+rudderanalytics.identify(
+  "1hKOmRA4GRlm", {
+    email: "alex@example.com"
+  }, {
+    integrations: {
+      Vero: {
+        tags: {
+          add: ["alex", "keener", "neworleans"],
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/vero/cloud-mode.mdx:165]:
```suggestion
          add: ["tag1", "tag2"],
```

File: docs/destinations/streaming-destinations/vero/device-mode.mdx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+```javascript
+rudderanalytics.identify(
+  "1hKOmRA4GRlm", {
+    email: "alex@example.com"
+  }, {
+    integrations: {
+      Vero: {
+        tags: {
+          add: ["alex", "keener", "neworleans"],
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/vero/device-mode.mdx:133]:
```suggestion
          add: ["tag1", "tag2"],
```

**Review by ItsSudip — APPROVED** [review]

File: docs/destinations/streaming-destinations/vero/cloud-mode.mdx [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+);
+```
+
+## Screen
+
+The <Link to="/event-spec/standard-events/screen/">`screen`</Link> call lets you record whenever your user views their mobile screen, with any additional relevant information about the screen.
+
+RudderStack sends the `screen` events to Vero via their [`events/track`](https://developers.getvero.com/?bash#events-track) endpoint.
+
+A sample `screen` call sent via the <Link to="/sources/sdk/rudderstack-android-sdk/">Android SDK</Link> is as shown:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/vero/cloud-mode.mdx:126]:
```suggestion
A sample `screen` call sent via the <Link to="/sources/sdks/rudderstack-android-sdk/">Android SDK</Link> is as shown:
```

File: docs/destinations/streaming-destinations/vero/cloud-mode.mdx [hunk 7]
```diff
@@ -0,0 +1,180 @@
+---
+title: "Cloud Mode"
+description: >-
+  Detailed technical documentation on sending events to Vero via the RudderStack cloud mode.
+---
+
+RudderStack lets you send your event data to Vero via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
+
+<div class="infoBlock">
+Find the open source transformer code for this destination in the <a href="">GitHub repository</a>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/vero/cloud-mode.mdx:10]:
```suggestion
Find the open source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/vero">GitHub repository</a>.
```

**Pull request merged**